src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioWorkspaceHost.cs

```diff
@@ -34,5 +34,6 @@ internal interface IVisualStudioWorkspaceHost
         void OnAdditionalDocumentOpened(DocumentId documentId, ITextBuffer textBuffer, bool isCurrentContext);
         void OnAdditionalDocumentClosed(DocumentId documentId, ITextBuffer textBuffer, TextLoader loader);
         void OnAdditionalDocumentTextUpdatedOnDisk(DocumentId id);
+        void OnIntellisenseBuildResult(ProjectId projectId, bool succeed);
```

Comment:
this require code lens change. should I change code lens or make it a separate interface?